tests/eas/load_tracking.py

```diff
@@ -45,6 +45,15 @@ class _LoadTrackingBase(LisaTest):
         'modules': ['cpufreq', 'cgroups'],
     }
 
+    @memoized
+    @staticmethod
+    def _get_cpu_capacity(test_env, cpu):
+        if test_env.nrg_model:
+            return test_env.nrg_model.get_cpu_capacity(cpu)
+        else:
```

Comment:
Actually... I went to edit this and it's harder to read IMO:

```
        if test_env.nrg_model:
            return test_env.nrg_model.get_cpu_capacity(cpu)
        return test_env.target.read_int(
            '/sys/devices/system/cpu/cpu{}/cpu_capacity'.format(cpu))
```
vs
```
        if test_env.nrg_model:
            return test_env.nrg_model.get_cpu_capacity(cpu)
        else:
            return test_env.target.read_int(
                '/sys/devices/system/cpu/cpu{}/cpu_capacity'.format(cpu))
```
In the second one you can easily see "this either returns X or Y" but in the first one you have to be like "if P then it returns X, then if that hasn't happened it returns Z"... do you know what I mean?